lib/plugins/entry-files-manifest.js

```diff
@@ -9,26 +9,41 @@
 
 'use strict';
 
-const EntryFilesManifestPlugin = require('../webpack/entry-files-manifest-plugin');
 const PluginPriorities = require('./plugin-priorities');
-const path = require('path');
 const sharedEntryTmpName = require('../utils/sharedEntryTmpName');
-const manifestKeyPrefixHelper = require('../utils/manifest-key-prefix-helper');
+const AssetsPlugin = require('assets-webpack-plugin');
+
+function processOutput(assets) {
+    for (const asset in assets) {
+        if (sharedEntryTmpName === asset) {
+            delete assets[asset];
```

Comment:
Couldn't this be an unconditionnal `delete assets[sharedEntryTmpName];` placed outside of the loop instead?